dali/operators/sequence/optical_flow/turing_of/optical_flow_turing.h

```diff
@@ -18,6 +18,7 @@
 #include <cuda_runtime.h>
 #include <memory>
 #include <string>
+#include <functional>
```

Comment:
Used in line 146

Reply:
Where this problem has surfaced?

Reply:
#2948, where I removed almost everything from backend.h